src/TTFontFile.php

```diff
@@ -3646,7 +3646,9 @@ function makeSubset($file, &$subset, $TTCfontID = 0, $debug = false, $useOTL = f
 			$glyphSet[$originalGlyphIdx] = $n; // old glyphID to new glyphID
 			$n++;
 		}
-
+		// necessary for using with fonts like font awesome. in some cases, variable was not defined
+		// error message was: Notice: Undefined variable: codeToGlyph
```

Comment:
The information in the comment is now included in the commit messages, it isn't? and it is not required for future reading of the code. 
Could you remove this comment?

Reply:
OK, I removed the comment.